binding.gyp

```diff
@@ -38,6 +38,23 @@
 					} ]
 				},
 				{
+					"target_name": "copyapis",
+					"type": "none",
+					"conditions": [ [ "'<!(echo $BUILD_NPM_PACKAGE)'=='y'", {
+						"actions": [ {
+							"action_name": "copyapis",
+							"message": "Copying JS APIs",
+							"outputs": [ "./uart.js" ],
```

Comment:
OTOH, if the support is not there, it doesn't matter if we copy the corresponding high-level APIs or not, because applications not taking these things into account will fail anyway. Whether they fail because the low-level soletta object lacks the appropriate bindings or whether they fail because
```JS
require( "soletta/uart" );
```
fails, it doesn't matter. They fail.